### ISSUE TITLE
Desklet Bug

### ISSUE THREAD

**Issue by counterpoint** [linuxmint/cinnamon-spices-desklets#1197]
### Desklet version/Build date

Version 1.6 (2024-04-18 13:21:08)

### Cinnamon version

6.0.4

### Distribution

Mint 21.3

### Graphics hardware *and* driver used

Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics]

### Desklet name and maintainer

calendar@schorschii @schorschii

### What happened?

Count of appointments reflects individual appointments. But it ignores recurring appointments. For example, today I have created a sample appointment, but also have two regular appointments. The desklet shows 1, where it should show 3.

**Comment by schorschii** [comment]
That's correct, recurring appointments are not implemented. Are you using this actively or did you discover this by chance? I'm considering removing the feature. It would be much work to implement this completely. It's not only daily, weekly, monthly, weekly - custom rules like "only Monday & Tuesday every 2 weeks" are possible too in the ics/ical format. Also, my impression is that this feature is barely used by only a few users.
